lib/src/app/app.dart

```diff
@@ -12,61 +12,77 @@ typedef RobotPartLogPage = GetRobotPartLogsResponse;
 /// gRPC client for connecting to Viam's App Service
 ///
 /// All calls must be authenticated.
+///
+/// For more information, see [Fleet Management API](https://docs.viam.com/appendix/apis/fleet/).
 class AppClient {
   final AppServiceClient _client;
 
   AppClient(this._client);
 
   /// Get the id of the user with the email provided
+  ///
+  /// For more information, see [Fleet Management API](https://docs.viam.com/appendix/apis/fleet/).
   Future<String> getUserIdByEmail(String email) async {
     final request = GetUserIDByEmailRequest()..email = email;
     final GetUserIDByEmailResponse response = await _client.getUserIDByEmail(request);
     return response.userId;
   }
 
   /// Create a new [Organization]
-  Future<Organization> createOrganization(String name) async {
```

Comment:
Looks like this was removed by accident?

Reply:
yep - sorry